Library/Homebrew/test/dev-cmd/cat_spec.rb

```diff
@@ -15,4 +15,12 @@
       .and not_to_output.to_stderr
       .and be_a_success
   end
+
+  describe "when a source file isn't found" do
```

Comment:
I think in this case we shouldn't add an entire integration test just for this change. They're really slow and since this is just a messaging issue I think it's fine for it to be missed by tests.

Thanks for writing it, though!

Reply:
Good to go once this test is removed.